src/App.css

```diff
@@ -20,3 +20,64 @@
     padding-right: 8%;
     font-size: 5.0rem;
   }
+  .allBtn {
+    display: grid;

```

Comment:
It's really cool you used CSS grid. 
What resources did you use to do this or did you already know?